MediaBrowser.Providers/Tmdb/Models/Search/MovieResult.cs

```diff
@@ -2,11 +2,11 @@ namespace MediaBrowser.Providers.Tmdb.Models.Search
 {
     public class MovieResult
     {
-            /// <summary>
-            /// Gets or sets a value indicating whether this <see cref="TmdbMovieSearchResult" /> is adult.
-            /// </summary>
-            /// <value><c>true</c> if adult; otherwise, <c>false</c>.</value>
-            public bool Adult { get; set; }
+        /// <summary>
+        /// Gets or sets a value indicating whether this <see cref="MovieResult" /> is adult.
+        /// </summary>
+        /// <value><c>true</c> if adult; otherwise, <c>false</c>.</value>
+        public bool Adult { get; set; }
```

Comment:
Indentation seems off.

Reply:
The original file has a 8 space tab and 1 4 space tab, should I reformat the whole file?

Reply:
Yeah, just reformat it so it conforms to our standards.